org.sil.Bloom.yml

```
@@ -0,0 +1,647 @@
+# Bloom Flatpak package build specification
+#
+# Copyright (c) 2022 SIL International. MIT License.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+app-id: org.sil.Bloom
+
+runtime: org.gnome.Platform
+runtime-version: "41"
+sdk: org.gnome.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.node16
+    - org.freedesktop.Sdk.Extension.mono6
+writable-sdk: true # Required to install msbuild for building libpalaso and geckofx
+
+# maybe someday, build a custom ffmpeg with just what we want
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: .
+
+finish-args:
+    - --socket=x11
+    - --share=ipc
+    - --share=network
+    - --socket=pulseaudio
+    # Bloom can import books and share folders over networks (either local or
+    # via a mechanism like DropBox).  Users need to load pictures, sound files,
+    # and videos from arbitrary locations in the filesystem. Users also need to
+    # be able to export books in various formats to arbitrary locations.
+    - --filesystem=home
+    - --filesystem=host
+    - --filesystem=xdg-data
+    # Bloom writes temporary folders and files to /tmp, to open in a browser to
+    # show the user, such as creating PDFs, ePUBs, or BloomPub ebooks.  Use the
+    # same /tmp as the host, so the host programs can find the files.
+    - --filesystem=/tmp
+    # libGL needs this
+    - --device=all
```

actually it only needs
```suggestion
    # libGL needs this
    - --device=dri
```

---

changed this as suggested